src/components/search/results/resultsList/ResultItem/ResultItem.tsx

```diff
@@ -39,16 +44,18 @@ export const ResultItem: FC<ResultItemProps> = observer(({ hit, url, index }) =>
     const isPreview = hit._collection === hashState.preview?.c && hit._id === hashState.preview.i
     const unsearchable = !!hashState.preview
 
-    const nodeRef = useRef()
+    const nodeRef = useRef<CustomDiv>(null)
     const handleMouseDown = (): void => {
-        ;(nodeRef.current as any).willFocus = !((nodeRef.current as any).tUp && timeMs() - (nodeRef.current as any).tUp < 300)
+        if (!nodeRef.current) return
+        nodeRef.current.willFocus = !(nodeRef.current.tUp && timeMs() - nodeRef.current.tUp < 300)
     }
     const handleMouseMove = (): void => {
-        ;(nodeRef.current as any).willFocus = false
+        if (!nodeRef.current) return
+        nodeRef.current.willFocus = false
     }
     const handleMouseUp = (): void => {
-        if ((nodeRef.current as any).willFocus) {
-            ;(nodeRef.current as any).tUp = timeMs()
+        if (nodeRef.current?.willFocus) {
+            nodeRef.current.tUp = timeMs()
             openPreview(hit)
         }
     }
```

Comment:
maybe refactor this to store `willFocus` and `tUp` in state?

Reply:
I am not against refactoring this peace of code, but it should be done in separate task. Considering this one was all about removing any types.